conversions/octal_to_binary.py

```diff
@@ -0,0 +1,22 @@
+def octal_to_binary(octal_number: str) -> str:
```

Comment:
provide description or a wikipedia link so that viewer cn understand aboit this octal and binary terms.

Also add doctests with unqiue edge cases.

Reply:
check!